.github/workflows/lint-test.yml

```diff
@@ -17,8 +17,12 @@ jobs:
       - uses: jpribyl/action-docker-layer-caching@v0.1.0
         # Ignore the failure of a step and avoid terminating the job.
         continue-on-error: true
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          install: true
       - name: Build Precovery in Docker
-        run: docker build -t precovery:$IMAGE_TAG .
+        run: docker build --load -t precovery:$IMAGE_TAG .
       - name: isort check
         run:  docker run -i precovery:$IMAGE_TAG isort --check-only .
```

Comment:
These are going to fail because we removed the packages from `setup.cfg`. Instead, it now relies on `pre-commit` running (I don't think dependencies get installed until you perform a run?). So we can reduce all these steps to a single `pre-commit run --all-files` or we can do something else.

Reply:
Yep, good call, those should be removed. The pre-commit dependencies get installed in a separate virtualenv anyway so they aren't accessible in this way.